README.md

```diff
@@ -0,0 +1,66 @@
+# ElitePad Subgrpah
+
+## 1. setup contract enviroment
+1. clone contract project and switch to 'fullTest' branch.
+```
+git clone https://github.com/nft-squared/whitelist-contracts
+cd whitelist-contracts
+git checkout origin/fullTest -b fullTest
+```
+2. install: `npm insall` or `yarn`
+3. start a local evm node: `yarn enode`
+4. deploy contracts and do some test: `yarn test --network develop test/FullLogic.test.ts`
```

Comment:
yarn compile

Reply:
@wfubuild Actually typechain is the auto-generated data from `yarn compile`. Shall always keep typechain in gitignore file.